### PR TITLE
Allows to specify max number of votes in a poll

### LINKED ISSUE #187: Define specific number of available votes for poll
Polls should have an option to define how many votes a user can give - instead of the current "multiple choice or not".

### PR DESCRIPTION
**Pull request by Nef10**
fix #187

### CODE REVIEW

File: _1327/polls/templates/polls_results.html [hunk 1]
```diff
@@ -14,6 +14,11 @@
             </div>
         </div>
     {% endif %}
+	{%  if poll.max_allowed_number_of_answers > 1 %}
+		<div class="alert alert-info" role="alert">
+			{% blocktrans with max_allowed_number_of_answers=poll.max_allowed_number_of_answers %}Every participant had up to {{ max_allowed_number_of_answers }} votes.{% endblocktrans %}
```
Comment (Bartzi) [inline _1327/polls/templates/polls_results.html]: shouldn't it be: Every participant **has** up to {{ max_allowed_number_of_answers }} votes.

**Comment by Bartzi** [comment]
:+1:

**Comment by invliD** [comment]
Please squash 52aecc5 and e85788b and add a meaningful commit message to that new commit.

**Comment by invliD** [comment]
:+1:

**Pull request merged**